src/main/java/com/google/devtools/build/lib/exec/StandaloneTestStrategy.java

```diff
@@ -181,7 +181,14 @@ public StandaloneTestStrategy(
       // returning the last list?
       return standaloneTestResult.spawnResults();
     } catch (IOException e) {
-      actionExecutionContext.getEventHandler().handle(Event.error("Caught I/O exception: " + e));
+      // Print the stack trace, otherwise the unexpected I/O error is hard to diagnose.
+      // A stack trace could help with bugs like https://github.com/bazelbuild/bazel/issues/4924
+      StringBuilder sb = new StringBuilder();
+      sb.append("Caught I/O exception:");
```

Comment:
I know that the message is mostly "io error", but I'd add it to the output anyway. Wdyt?

Reply:
Sorry, I don't understand what is it you're suggesting, could you please clarify?

Reply:
I mean, I don't understand what action you ask of me.

Reply:
I'm sorry for not being clearer. I'm asking you to add e.getMessage() to the sb.

Reply:
I see! Done.